test/cypress/e2e/integration/app-frontend/attachments-in-group.js

```diff
@@ -430,7 +430,6 @@ describe('Repeating group attachments', () => {
 
     // Verify that one of the attachments in the next nested row is visible in the table header. This is also a trick
     // to ensure we wait until the deletion is done before we fetch the redux state.
-    cy.get(appFrontend.group.rows[0].nestedGroup.rows[0].editBtn).click();
```

Comment:
Maybe add tests for the new behaviour where deleting a single row with `openByDefault` now leaves the group closed for editing?

Reply:
I agree, but having assertions for this in the test won't hurt! You changed the behaviour of something, and since this is the only part that changes any of the cypress tests, it looks to me like we don't have enough test coverage to clearly define the behaviour of `openByDefault` in our tests.

Reply:
Maybe, the tests do cover all of the documented behaviour, but it was not supposed to do anything when a row is deleted as long as there are more rows left.